docroot/modules/custom/va_gov_post_api/src/Service/PostFacilityService.php

```diff
@@ -161,7 +170,16 @@ public function queueFacilityService(EntityInterface $entity, bool $forcePush =
         // endpoint.
         if (!empty($data['payload']) && !empty($facilityApiId)) {
           $this->postQueue->addToQueue($data, $this->shouldDedupe());
-          $this->logService($facilityApiId, $this->facilityService);
+          if ($data['payload']['detailed_services']['0'] && (self::$logFile)) {
```

Comment:
```suggestion
          if (!empty($data['payload']['detailed_services'][0]) && (self::$logFile)) {
```
Two things  :
I think the 0 is not a string
We want to make the !empty just in case 0 does not exist.  Reaching for a 0 that is not there will throw a notice.